### ISSUE TITLE
[Feature Request] Move Entries

### ISSUE THREAD

**Issue by NWuensche** [lfos/calcurse#136]
It would be nice to have a way to move an entry without copy-paste and deleting it. For repeating tasks, there is a feature to edit the end date. Moving an entry could work in the same manner.

**Comment by lhca** [comment]
Are you using version 4.3.0?

Instead of copy-paste you can delete-paste. Moving an item is already possible: just edit an item and choose (5) Move.

**Comment by NWuensche** [comment]
I'm using 4.2.1, because that's the latest version for `apt` on Ubuntu 18.04. Is it possible to update this? I would also do this if it is possible.

**Comment by NWuensche** [comment]
So I built 4.3.0. I can delete-paste now, but when I edit an Item with "e", I just can edit the description, not the date. I don't really know what you mean by (5) Move. I cannot select any 5th option in edit screen.

**Comment by lhca** [comment]
Items are either appointments with start and end time or events without them (an "all-day event").

If the item is an appointment the edit item command has four subcommands: (1) start time, (2) end time, (3) description and (4) move (if the item is a repeated appointment there are five). If the item is an event there is just one possibility: edit the description.

It seems you have added an event.

**Comment by NWuensche** [comment]
Yep, I added an event. However, is there a reason you cannot move events?

**Comment by lhca** [comment]
None. It's just not implemented. Moving appointments was new in 4.2.0, I believe.

**Comment by NWuensche** [comment]
I will try to add this. However, I can't get the GitHub branch working. After autogen and configure, I get this while executing make:

```
Making all in doc
make[2]: Verzeichnis „/home/nwuensche/calcurse/doc“ wird betreten
make[2]: *** Keine Regel vorhanden, um das Ziel „calcurse.1“, 
  benötigt von „all-am“, zu erstellen.  Schluss. 
make[2]: Verzeichnis „/home/nwuensche/calcurse/doc“ wird verlassen
make[1]: *** [Makefile:388: all-recursive] Fehler 1
make[1]: Verzeichnis „/home/nwuensche/calcurse“ wird verlassen
make: *** [Makefile:329: all] Fehler 2
```
So I have no rule for creating `calcurse.1`. 

This is the output from configure:

```
========================================================================
calcurse is configured as follows.
Please check that this configuration matches your expectations.
Also give a look at the config.h file to check for preprocessor symbols.

Host system type   : x86_64-pc-linux-gnu

Options used to compile and link:
  PREFIX           = /usr/local
  VERSION          = 4.3.0-63-gea0a
  CC               = gcc
  CFLAGS           = -g -O2 -Wall
  CPPFLAGS         = 
  DEFS             = -DHAVE_CONFIG_H
  LD               = /usr/bin/ld
  LDFLAGS          = 
  LIBS             =  -lncursesw -pthread -lm
========================================================================

```

**Comment by lhca** [comment]
The input format is user-configurable. The default format is, I think, the US-american mm/dd/yyyy. You set it in the general options menu (type `C` then `g`).

A more accommodating prompt is upcoming in the next version, soon to be released.

**Comment by Ierezell** [comment]
@NWuensche You need to have all the requirements. 

gcc
automake
asciidoc
gettext
ncurses

I get the same problem on archlinux but just installing asciidoc will solve it. 

just copy paste : 
For debian 
`sudo apt-get install gcc automake asciidoc gettext ncurses`
For arch 
`sudo pacman -S gcc automake asciidoc gettext ncurses`
Or something else depending of your distro..

**Comment by NWuensche** [comment]
@Ierezell Thank you so much. I was really missing asciidoc.
